lmdb.zig

```diff
@@ -123,8 +123,8 @@ pub const Environment = packed struct {
             try call(c.mdb_env_copy2, .{ self.inner, backup_path.ptr, flags.into() });
         }
     }
-    pub inline fn pipeTo(self: Self, fd: os.fd_t, flags: CopyFlags) !void {
-        try call(c.mdb_env_copyfd2, .{ self.inner, fd, flags.into() });
+    pub inline fn pipeTo(self: Self, fdHandle: os.fd_t, flags: CopyFlags) !void {
```

Comment:
Keep all variable names except for struct and function names snake-cased. Also, does a compile error happen if the parameter name is unchanged?

```suggestion
    pub inline fn pipeTo(self: Self, fd_handle: os.fd_t, flags: CopyFlags) !void {
```

Reply:
Yes, it was complaining about the parameter shadowing the function on line 240.